src/operator/MigrateOrgOverlay.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {FC, useContext} from 'react'
+import {
+  Alert,
+  ButtonBase,
+  ButtonShape,
+  ComponentColor,
+  ComponentStatus,
+  Gradients,
+  IconFont,
+  Overlay,
+  RemoteDataState,
+} from '@influxdata/clockface'
+import {OperatorAccount} from 'src/client/unityRoutes'
+import {OverlayContext} from './context/overlay'
+
+interface Props {
+  toAccount: OperatorAccount
+}
+
+const MigrateOrgOverlay: FC<Props> = ({toAccount}) => {
+  const {
+    handleMigrateOrg,
+    migrateOverlayVisible,
+    migrateStatus,
+    organization,
+    setMigrateOverlayVisible,
+  } = useContext(OverlayContext)
+
+  const migrateOrg = () => {
+    try {
+      handleMigrateOrg(organization.idpeId, toAccount.id.toString())
+    } catch (e) {
+      setMigrateOverlayVisible(false)
+    }
+  }
+
+  const message = `
+    This action will migrate the following organization and users 
+    from ${organization.account.id ?? 'N/A'} to 
+    ${toAccount?.name} (${toAccount?.id})`
```

Comment:
Will this only fire with non-OSS? I'm guessing so since, like you said, there is other functionality that is part of this shared codebase, but this one stuck out since OSS has orgs (as opposed to a connection to quartz, accounts, etc)